iwf/state_options.go

```diff
@@ -5,12 +5,16 @@ import (
 )
 
 type StateOptions struct {
-	SearchAttributesLoadingPolicy *iwfidl.PersistenceLoadingPolicy
-	DataAttributesLoadingPolicy   *iwfidl.PersistenceLoadingPolicy
-	WaitUntilApiTimeoutSeconds    *int32
-	ExecuteApiTimeoutSeconds      *int32
-	WaitUntilApiRetryPolicy       *iwfidl.RetryPolicy
-	ExecuteApiRetryPolicy         *iwfidl.RetryPolicy
-	WaitUntilApiFailurePolicy     *iwfidl.WaitUntilApiFailurePolicy
-	ExecuteApiFailureProceedState WorkflowState
+	WaitUntilApiSearchAttributesLoadingPolicy *iwfidl.PersistenceLoadingPolicy
```

Comment:
The comments should be here, just lie Java docs

Reply:
Ah thanks! Didn't see this file at first. Done now ✅